README.md

```diff
@@ -1,2 +1,288 @@
-# distributed-press-cli
-Command Line Interface for the Distributed Press API and Social Inbox
+# Distributed Press CLI
+
+A **Command Line Interface (CLI)** for interacting with the Distributed Press API and Social Inbox. Manage your sites, register them with Social Inbox, and engage with followers seamlessly from the command line.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Usage](#usage)
+  - [Create a New Site](#create-a-new-site)
+  - [Register Site with Social Inbox](#register-site-with-social-inbox)
+  - [Send a Post to Followers](#send-a-post-to-followers)
+- [Commands](#commands)
+- [Examples](#examples)
+- [License](#license)
+- [Resources](#resources)
+
+## Installation
```

Comment:
These sound like `development` instructions. I'll publish the repo to NPM and our instructions should either use `npx dp-cli` or use `npm i -g dp-cli`